.github/workflows/ci.yml

```diff
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches: [ '*' ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+
+  unit-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18
+
+    - name: Build
+      run: go build -v cmd/webhook/main.go
+    - name: pretest
+      run: cd test/pretest &&  ./env_setup.sh
+    - name: Test
+      run: go test -v ./...
+
+  e2e-test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.18
+
+    - name: modify host
+      run: cd test/pretest && sudo ./modify_host.sh
+    - name: pretest
+      run: cd test/pretest &&  ./env_setup.sh
+    - name: run e2e test
+      run: cd test && ./test.sh
+  
```

Comment:
Only one empty line at the end

Reply:
@ComradeProgrammer still not right

![image](https://user-images.githubusercontent.com/3787410/180449318-6596408e-981e-4b00-a780-2db2f11f13ac.png)